src/main/kotlin/io/snyk/plugin/ui/toolwindow/VulnerabilityTreeCellRenderer.kt

```diff
@@ -106,8 +105,18 @@ class VulnerabilityTreeCellRenderer : ColoredTreeCellRenderer() {
                     issue.title
                 } + if (issue.ignored) " (ignored)" else ""
 
+                var obsolete = false
+                if (value.parent is IacFileTreeNode) {
+                    val parentTreeNode = value.parent as IacFileTreeNode
+                    val iacIssuesForFile = parentTreeNode.userObject as IacIssuesForFile
+                    if (iacIssuesForFile.obsolete) {
+                        text += obsoleteSuffix
```

Comment:
for OSS and Code we mark FileNode with ` (obsolte)` text, make it gray and make childern (Issues) gray. Shell we do the same here for consistancy or change other products instead?

Reply:
It does it, too - where do you see the difference?

Reply:
here it mark Issues as (oboslete) instead of FileNode